src/v2.rs

```diff
@@ -269,23 +269,59 @@ pub struct Aggregator {
 }
 
 impl Aggregator {
-    /// Check and aggregate the party signatures
+    /// Aggregate the party signatures using a tweak
     #[allow(non_snake_case)]
     pub fn sign_with_tweak(
         &mut self,
         msg: &[u8],
         nonces: &[PublicNonce],
         sig_shares: &[SignatureShare],
-        key_ids: &[u32],
+        _key_ids: &[u32],
```

Comment:
Did you decide against changing the function signature to avoid a major release?

Reply:
Yes exactly.